addons/l10n_sa/README.md

```diff
@@ -0,0 +1,12 @@
+dvit-odoo8
+===========
+
+Odoo Arabic localization for most arabic countries and Saudi Arabia.
```

Comment:
Don't create a README file, put it in the `description` section of `__openerp__.py` instead